lib/concurrent/utility/native_extension_loader.rb

```diff
@@ -14,6 +14,10 @@ module NativeExtensionLoader
       def allow_c_extensions?
         Concurrent.on_cruby?
       end
+   
+      def loaded_c_extensions?
+        NativeExtensionLoader.instance_variable_get(:@c_ext_loaded)
```

Comment:
I think instance_variable get might be needed because ``` @c_ext_loaded ``` is nil in the context of the ``` Concurrent ``` module and it is true in the context of the ``` 
NativeExtensionLoader ``` module.
```ruby
module Concurrent
  module Utility
    module NativeExtensionLoader
      def c_ext_loaded
        @c_ext_loaded
      end
      def c_ext_loaded_with_iv_get
        NativeExtensionLoader.instance_variable_get(:@c_ext_loaded)        
      end
    end
  end
 ～～～～～～～～
 ～～～～～～～～～
end

Concurrent.c_ext_loaded? # => nil
Concurrent.c_ext_loaded_with_iv_get # => true
```